docroot/profiles/lagunita/sul_profile/modules/sul_helper/src/Layouts/SulThreeColumn.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Drupal\sul_helper\Layouts;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\stanford_layout_paragraphs\Layouts\ThreeColumn;
+
+/**
+ * Three column layout class
+ */
+class SulThreeColumn extends ThreeColumn  {
+  use LayoutWithHeading;
+  
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
```

Comment:
Needs docblock
```suggestion
  /**
   * {@inheritdoc}
   */
  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
```

same for other methods.